tripy/tests/frontend/test_shape.py

```diff
@@ -511,7 +511,7 @@ def test_quantize_rejected(self, values):
     def test_binary_elementwise_broadcast_rejected(self, values):
         with raises(
             tp.TripyException,
-            match=r"\_\_mul\_\_ expects tensor arguments to have matching class types, but got mixed `tp\.Tensor` and `tp\.Shape` arguments\.",
+            match=r"Error processing shape inputs in operator BinaryElementwise Further information: Binary elementwise operators do not accept combinations of Shape and Tensor arguments.",
```

Comment:
```suggestion
            match=r"Error processing shape inputs in operator BinaryElementwise. Further information: Binary elementwise operators do not accept combinations of Shape and Tensor arguments.",
```

Reply:
@jhalakpatel I'm going to replace the part of the code that emits this error as part of the shape refactoring. Will fix the error messages then.